src/test/regress/sql/multi_table_ddl.sql

```diff
@@ -122,6 +122,13 @@ ALTER TABLE test_table ALTER COLUMN id3 SET DEFAULT nextval('test_sequence_1'),
 ALTER TABLE test_table ADD COLUMN id3 bigserial;
 ALTER TABLE test_table ADD COLUMN id4 bigserial CHECK (id4 > 0);
 
+CREATE SEQUENCE pg_temp.temp_sequence;
+CREATE TABLE table_with_temp_sequence (
```

Comment:
Citus local table creation produces a weird error, can we use similar to distributed/reference tables? Maybe opening issue is fine for now
```SQL
show citus.use_citus_managed_tables;
┌────────────────────────────────┐
│ citus.use_citus_managed_tables │
├────────────────────────────────┤
│ on                             │
└────────────────────────────────┘
(1 row)

Time: 3.232 ms
[local] onderkalaci@postgres:5432-71894=# create table pg_temp.test(a int);
ERROR:  constraints on temporary tables may reference only temporary tables
Time: 31.384 ms

```

Reply:
The reason behind this error message was the discussion made here: https://github.com/citusdata/citus/pull/4592#discussion_r565397368

However, now that citus 11 allows using citus local tables for more than just foreign keys, this message needs to be changed as well.

Reply:
https://github.com/citusdata/citus/issues/5841